### DISCUSSION TITLE
Rancher Desktop 1.6.1

### DISCUSSION THREAD

**Discussion by jandubois** [rancher-sandbox/rancher-desktop#3266] [General]
This is the 1.6.1 release of Rancher Desktop, an open source desktop application to bring Kubernetes and container management to Mac, Windows, and Linux.

## This patch release fixes 3 regressions in the 1.6.0 release

* On Windows, the `host.rancher-desktop.internal` and `host.docker.internal` names did not resolve.

* On macOS, the bridged interface was not being used, even when running with administrative rights, and only port forwarding to localhost was active.

* On macOS and Linux, upgrades from a previous release did not update all files inside the VM. In particular, `nerdctl` would stay at the previous version (e.g. `0.22.2` in Rancher Desktop 1.5.1) instead of being upgraded to `0.23.0`.

## Connect with the developers

- The [issue queue](https://github.com/rancher-sandbox/rancher-desktop/issues)

- [Rancher Users Slack](https://slack.rancher.io/) in the #rancher-desktop channel

## Changelog

The [full version changelog, from v1.6.0, can be found using GitHub compare](https://github.com/rancher-sandbox/rancher-desktop/compare/v1.6.0...v1.6.1) and the details of the release can be found in the [v1.6.1 milestone](https://github.com/rancher-sandbox/rancher-desktop/milestone/26?closed=1).

<hr /><em>This discussion was created from the release <a href='https://github.com/rancher-sandbox/rancher-desktop/releases/tag/v1.6.1'>Rancher Desktop 1.6.1</a>.</em>

**Comment by mfougeron** [comment]
Since update on macOS 13 followed by Rancher 1.4.1 -> 1.6.1, dashboard is infinite loading.
Also, diagnostic says : 
"The ~/.rd/bin directory has not been added to the PATH, so command-line utilities are not configured in your zsh shell."
Even after adding ~/.rd/bin manually to the PATH.
Tried multiple Rancher and computer restarts but still the same...
